vendor/k8s.io/kubernetes/pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -95,7 +94,6 @@ func NewCmdRollingUpdate(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	}
 	cmd.Flags().Duration("update-period", updatePeriod, `Time to wait between updating pods. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".`)
 	cmd.Flags().Duration("poll-interval", pollInterval, `Time delay between polling for replication controller status after the update. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".`)
-	cmd.Flags().Duration("timeout", timeout, `Max time to wait for a replication controller to update before giving up. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".`)
```

Comment:
@fabianofranz I am unsure how we would handle cases where timeout has a specific description (or default value) for a command. For example, [newtoken.go][(https://github.com/openshift/origin/blob/master/pkg/cmd/cli/sa/newtoken.go#L87) has a timeout of `30s`.